rke/k8s_rke_system_images.go

```diff
@@ -4332,6 +4332,42 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.6"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band post v2.4.15
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		// Reminder: This template should remain an exact copy of the template for v2.5.x (No ACI images)
```

Comment:
Updated this comment to something more appropriate since the previous one wasn't accurate anymore